browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -317,9 +318,14 @@ test.describe(
       )
     })
 
-    test('categorizes programs for draft and applied applications as guest user', async () => {
+    test('categorizes programs for draft and applied applications', async () => {
       const {applicantQuestions, page} = ctx
 
+      // TODO (https://github.com/civiform/civiform/issues/7377): Remove once we have a header with login/logout.
```

Comment:
**suggestion:** Would you mind changing it to this format here and below: `TODO(#7377): Remove..`

Reply:
Done, thanks!